.gitignore

```diff
@@ -0,0 +1,8 @@
+# common
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+
+docs/
```

Comment:
can you elaborate the need of this?

Reply:
This repo has two branches, the `Releases` branch which contains the Swift Package file, along with the `README.md` and other docs.

The other branch is the `gh-pages` branch which is where our public API docs are hosted. If you switch between branches locally, you may end up with the `docs/` folder from the `gh-pages` branch. In an effort to reduce human error, we can ignore the `docs/` folder when working in the other branches of this repo.

Reply:
And the other items are from a standard Mac ignore list that I took from our internal code repo to squash any other hidden files that may show up when working locally on these docs.

Reply:
sounds good.